CMakeLists.txt

```diff
@@ -79,6 +79,28 @@ endif()
 
 find_package(BLAS REQUIRED)
 find_package(LAPACK REQUIRED)
+find_package(OpenMP REQUIRED)
```

Comment:
In the future it would be nice to have OpenMP as an option. For some reason my code "option(OPENMP "Enable OpenMP support" ON)" related to the feature is removed.

Reply:
see below